app/views/layouts/application.html.erb

```diff
@@ -29,6 +29,7 @@
       <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
I think the yield commands will help push some of this code out of
application.html.erb and back into the specific pages--it will be a
backward-compatible change, and we can follow up with some cleanup
submissions, either by the "original" page authors or others who become
familiar with how the javascript is used.

On Fri, Jul 29, 2016 at 1:17 PM, Jeffrey Warren <notifications@github.com>
wrote:

> In app/views/layouts/application.html.erb
> <https://github.com/publiclab/plots2/pull/633#discussion_r72828102>:
>
> > @@ -29,6 +29,7 @@
> >        <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
> >      <![endif]-->
> >
> > +    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
>
> At some point soon, @david-days <https://github.com/david-days> code will
> merge and he uses a yield command to enable subviews to add includes up
> top.
>
>
> https://github.com/david-days/plots2/blob/master/app/views/layouts/application.html.erb#L40
>
> If his merge is near enough, we could just live with the inefficiency
> until then. I'll leave a note to that effect in #624
> <https://github.com/publiclab/plots2/issues/624>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/633/files/ad3255c42e60c8fce2b5beec70c02a5ce2d90a7b#r72828102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOOfYI3m-AZFMDpGbWDvlAwhf1BPyC7Rks5qajXBgaJpZM4JTZ5B>
> .
>